packages/dds/merge-tree/src/test/client.rollback.spec.ts

```diff
@@ -51,6 +51,21 @@ describe("client.rollback", () => {
         const marker = client.getMarkerFromId("markerId");
         assert.notEqual(marker?.removedSeq, undefined);
     });
+    it("Should rollback insert and validate the partial lengths", () => {
+        client.insertTextLocal(0, "ghi");
+        client.insertTextLocal(0, "def");
+        client.insertTextLocal(0, "abc");
+
+        client.rollback?.({ type: MergeTreeDeltaType.INSERT }, client.peekPendingSegmentGroups());
+
+        assert.equal(client.getText(), "defghi");
+        validatePartialLengths(client.getClientId(), client.mergeTree);
+
+        client.rollback?.({ type: MergeTreeDeltaType.INSERT }, client.peekPendingSegmentGroups());
+
+        assert.equal(client.getText(), "ghi");
+        validatePartialLengths(client.getClientId(), client.mergeTree);
+    });
     it("Should rollback multiple inserts with split segments", () => {
```

Comment:
Would it be useful to validatePartialLengths in this test too?

Reply:
It most likely wouldn't hurt, I know that validatePartialLengths can be an expensive operation but calling it once here should be fine.